src/sagas/wallet.js

```diff
@@ -115,7 +116,6 @@ export function* startWallet(action) {
   const {
     words,
     pin,
-    fromXpriv,
```

Comment:
`fromXpriv` is not needed anymore since starting from the peristed access data is faster